hs_tracking/views.py

```diff
@@ -49,6 +50,18 @@ def get(self, request, **kwargs):
             msg = Variable.format_kwargs(**fields)
             session.record('app_launch', value=msg)
 
+            if settings.GEOHUB_HOMEPAGE_URL in url:
+                if settings.GEOHUB_HOMEPAGE_URL.endswith('/'):
```

Comment:
os.path.join should handle this case, making it one line.

Reply:
@sblack-usu This is the URL being passed in, so I don't think os.path.join can handle this, right? However, since GEOHUB_HOMEPAGE_URL will be moved from local_settings.py to settings.py, this check to see whether it ends with ```/``` is not needed. I will simplify it.